### ISSUE TITLE
Account page

### ISSUE THREAD

**Issue by scaspers** [DanielCaspers/CA-Portal#48]
Thoughts:

1. [x] Would it be better to replace pencil with 'edit' for the account page?
2. [x]  Add formatting (hyphens\make pretty) to phone number on account page?
3. [x]  Have edit\cancel\save all appear in same area?
4. [x]  Discuss having Phone#'s and emails in "cards" (edit/save/cancel/del for each).
5. [x]  Remove reference to thank you in VIP Rewards tool tips.

**Comment by DanielCaspers** [comment]
@scaspers  @YoMurphy ,

point 1) Open to changing. I decided on this pattern after playing around with a few mobile apps. I don't feel strongly on text vs icon button for this. I do feel that if we want full consistency, I would opt towards all text.

point 3) Open to changing. The reason I chose this inconsistency was because previously I got feedback from Steve that it may not be obvious why the form starts as disabled for some mobile users (especially those with very small viewport resolution height). The idea behind edit at the top is for quickly being able to see it and establish the pattern of how to enable the form. Putting the rest of the form controls at the bottom is because at that point the user is already moving through the form and as a result of reading it, finds the relevant actions for ending their edits either with save or cancel at the bottom.

point 4) I am open to this approach again, but I thought the same thing but got feedback that this might also be confusing in the current single-page format for all the data. If we feel strongly about changing this, I am going to insist we don't show all information at once, but instead force clickthrough of different tabs for address, email preferences, and phone preferences, similarly to how we use "tags" in DI inspection view to filter the data.

https://material.angular.io/components/tabs/examples

If you're both game for my suggestion for 4, I think that would also allow me to make better form validation user experience with the "Save" button enable and disable, which is currently really finicky with everything in one form. 

I took a stab at 2 and 5 and submitted it.
